src/templateAccessibilityAltTextRule.ts

```diff
@@ -0,0 +1,103 @@
+import { ElementAst, AttrAst, BoundElementPropertyAst, TextAst } from '@angular/compiler';
+import { sprintf } from 'sprintf-js';
+import { IRuleMetadata, RuleFailure, Rules } from 'tslint/lib';
+import { SourceFile } from 'typescript/lib/typescript';
+import { NgWalker } from './angular/ngWalker';
+import { BasicTemplateAstVisitor } from './angular/templates/basicTemplateAstVisitor';
+
+export class Rule extends Rules.AbstractRule {
+  static readonly metadata: IRuleMetadata = {
+    description: 'Ensures that all elements that require alternate text i.e. img, area, input[type="image"] and object.',
```

Comment:
Enforces alternate text for elements which require alt attribute

Reply:
Its not just alt attribute @mgechev, check the test cases